Project.swift

```diff
@@ -0,0 +1,57 @@
+import ProjectDescription
+
+let deploymentTarget: DeploymentTarget = .macOS(targetVersion: "10.15")
+let baseSettings: SettingsDictionary = ["EXCLUDED_ARCHS": "arm64"]
+
+let project = Project(name: "Tuist",
```

Comment:
Why do you think about different code styling.

```swift
let abc = ABC(
    foo: Foo(),
    bar: Bar()
)
```

instead of

```swift
let abc = ABC(foo: Foo(),
                        bar: Bar())
```

Reply:
it is just a small styling issue, but I much prefer the legibility of your suggestion @laxmorek - however I would leave that up-to the implementor to decide how it's formatted 😄

Reply:
Im in favor of #1 its more consistent and legible (:

Reply:
Since there seems for the first alternative (I prefer it as well), we could codify this by adding `--wraparguments before-first` to `.swiftformat`, so everyone can write it as they please but all code in main will use the same convention. If everybody is in favor, I'll create a PR today.

Reply:
I used to like the second option but switched to using the first one and it works better 99% of the time IMO